mobx_lint/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: mobx_lint
+description: A starting point for Dart libraries or applications.
+version: 1.0.0
+homepage: https://mobx.dev
```

Comment:
Need to update these properties with proper descriptions.

Reply:
@pavanpodila done.